### ISSUE TITLE
Add token usage information

### ISSUE THREAD

**Issue by rohitprasad15** [andrewyng/aisuite#50]
Token information received in response from the LLM is ignored.
Translate this information into the required format.

**Comment by CuriousCoder17** [comment]
@rohitprasad15 I'd love to contribute to this! Could you please assign to me?

**Comment by decodingafterlife** [comment]
Hey I'll love to contribute to this. Please assign it to me...

**Comment by tonykay** [comment]
Yes, this is a deal breaker currently for me, but if there is a path forward, I will happily try to adopt aisuite going forward.

**Comment by Sourabhbhalke** [comment]
i would like to contribute regarding token usage information.😃
I am actually working on token cost optimization project using aisuite and streamlit, where aisuite will help switching between models and test out different token cost optimization techniques.
currently, i have web-app codes here with and without aisuite
https://github.com/Sourabhbhalke/LiteLLMtests-public

**Comment by rohitprasad15** [comment]
Since [vishwamartur](https://github.com/vishwamartur) already started work on it. Assigning it to him.
Please tag me for review and any doubts. Code has changed since you raised the PR. Sorry for the inconvenience.. If possible please adapt to your code to the new code structure.
